src/content/1.7/release/release-a-new-version-of-prestashop.md

```diff
@@ -0,0 +1,83 @@
+2. we make it available on GitHub as a GitHub release, and tag the branch used for the build to mark the exact code commit corresponding to this build
+3. we make it available on prestashop.com as a downloadable ZIP archive
+4. we activate multiple settings in PrestaShop Addons system so the PrestaShop API allows the new version of PrestaShop to download modules adapted to it
+5. we activate multiple settings in PrestaShop Internationalization API so it allows the new version of PrestaShop to download localization packs
+6. we update the XML stream being used by auto-upgrade module to monitor what versions of PrestaShop are available
+7. we create new Docker images for the new version
+
+## 1. Make a build
+
+If you want to create a new version of PrestaShop from its branch `1.7.6.x`, here is the process:
+
+1. Clone the repository, checkout the branch
+2. Update the code to acknowledge the new version:
+ - update version number in files `app/AppKernel.php` and `nstall-dev/install_version.php` (see the 1.7.6.6 [example](https://github.com/PrestaShop/PrestaShop/pull/19980))
+ - update the Changelog file `docs/CHANGELOG.txt` and the `CONTRIBUTORS.md` files with new content and new contributors (see the 1.7.6.6 [example](https://github.com/PrestaShop/PrestaShop/pull/20032))
+3. Use the embedded CLI [PrestaShop Release Creator](https://github.com/PrestaShop/PrestaShop/blob/develop/tools/build/README.md) to create the ZIP archive
+
+Example of Release Creator usage: `php tools/build/CreateRelease.php --version="1.7.6.6"`
+
+This allows you to create a new ZIP archive which contains the software. It can be installed and run, however the installation process might be unable to download new language packs and download any modules from Addons if the Addons and Translation systems do not acknowledge this version.
+
+## 2. Make a new version on GitHub
+
+You need first to merge into the GitHub branch the work you have carried out for the Build: updating the version number and modifying Changelog and Contributors files.
+
+When it is merged, tag the last commit on this branch (not the merge commit, but rather the one before).
+
+Open GitHub and go to ["Releases" page](https://github.com/PrestaShop/PrestaShop/releases), click "Draft a new release".
+
+- use the tag you created as a target
+- use the Changelog as description
+- attach the ZIP archive
+
+And publish the new release !
+
+## 3. Make a new version on prestashop.com
+
+This information is private to PrestaShop company, but we can say it is simply a form where you input the version number and upload the new ZIP archive.
+
+## 4. Configure Addons API for the new version
+
+This information is private, but we can say it is simply a series of steps to create some artifacts which, when configured properly, allow the Addons API to serve modules for the new version.
+
+## 5. Configure Internationalization API for the new version
```

Comment:
Is there a link that we can add to explain how to build the localization packs from Crowdin?
Of course, the part about our API should stay secret, but if we can provide the more possible information about that part, it would be great.

Reply:
That's a big story 😄  I will try to add it later. It's a huge process and I am not sure I have all the keys.